app/views/install/node.html

```diff
@@ -23,7 +28,7 @@
     <div class="nhsuk-grid-column-two-thirds">
 
       <h1>
-        <span class="nhsuk-caption-l">Step 2 of 8</span>
+        <span class="nhsuk-caption-l">Step 2 of 8: {{system | default ("[add operating system with 'system']") }}</span>
```

Comment:
Might be safer to do nothing if we we forget to set `system` on a page?

```suggestion
        <span class="nhsuk-caption-l">Step 2 of 8{% if system %}: {{system }}{% endif %}</span>
```